bin/wait_for_it.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# little script that simply watches for the secret puller to be done.  
+# script will return 0 on success, 1 on timeout (we'll give the puller 60s), and 2 for any other error
+
+if [ -z ${DONE_FILE} ]; then
```

Comment:
if it picks the file based on it's own location we don't need to hardcode the `/secrets/` part